### PR TITLE
improving readability

### PR DESCRIPTION
**Pull request by tiagov8**

### CODE REVIEW

File: rules/do-you-use-aname-record/rule.md [hunk 1]
```diff
@@ -13,9 +13,15 @@ redirects: []
 
 ---
 
-What is ANAME record? ANAME record (also known as A record) is an alias record that allows you to map the apex record or any other record within your domain to a target host name, essentially a CNAME record for the apex record. ANAME record is especially useful for when you have multiple domain names and your website is hosted by a provider that changes it's IP Address, this does happen quite regularly with WPEngine. Many DNS service provider does not support ANAME record, however, [DNSMadeEasy](http://dnsmadeeasy.com/) has made this service available.
+ANAME record (also known as A record) is an alias record that allows you to map the apex record or any other record within your domain to a target host name, essentially a CNAME record for the apex record. 
+
+The ANAME record is especially useful for when you have multiple domain names and your website is hosted by a provider that changes it's IP Address, this does happen quite regularly with WPEngine. Many DNS service provider does not support ANAME record, however, [DNSMadeEasy](http://dnsmadeeasy.com/) has made this service available.
```
Comment (chrisschultzssw) [inline rules/do-you-use-aname-record/rule.md:18]: Change from
> Many DNS service provider does not support
To
Many DNS service providers do not support

**Review by chrisschultzssw — CHANGES_REQUESTED** [review]
Just one grammar thing :)

File: rules/do-you-use-aname-record/rule.md (hunk 1 above)
Comment (bradystroud) [reply rules/do-you-use-aname-record/rule.md:18]:
```suggestion
The ANAME record is especially useful for when you have multiple domain names and your website is hosted by a provider that changes it's IP Address, this does happen quite regularly with WPEngine. Many DNS service providers do not support ANAME record, however, [DNSMadeEasy](http://dnsmadeeasy.com/) has made this service available.
```

**Review by adamcogan — APPROVED** [review]

**Review by chrisschultzssw — APPROVED** [review]
👍

**Pull request merged**